pkg/discovery/processor/turbo_policy_processor.go

```diff
@@ -46,17 +53,23 @@ func (p *TurboPolicyProcessor) ProcessTurboPolicies() {
 	glog.V(2).Infof("Discovered %v PolicyBindings.", len(turboPolicyBindings))
 
 	policyMap := make(map[string]*repository.TurboPolicy)
+
 	for _, sloScale := range turboSloScalings {
-		// Create a copy as sloScale variable is reused during range loop
-		sloScaleCopy := sloScale
-		gvk := sloScaleCopy.GetObjectKind().GroupVersionKind()
-		if gvk.Empty() {
-			continue
+		if gvk := sloScale.GetObjectKind().GroupVersionKind(); !gvk.Empty() {
```

Comment:
Please see if you can refactor the code inside the two loops here

Reply:
will try to address this in https://vmturbo.atlassian.net/browse/OM-98560.